modules/simple-payments/simple-payments.php

```diff
@@ -377,5 +404,173 @@ function setup_cpts() {
 		register_post_type( self::$post_type_product, $product_args );
 	}
 
+	/**
+	 * Format a price for display
+	 *
+	 * Largely taken from WordPress.com Store_Price class
+	 *
+	 * The currency array will have the shape:
+	 *   format  => string sprintf format with placeholders `%1$s`: Symbol `%2$s`: Price.
+	 *   symbol  => string Symbol string
+	 *   desc    => string Text description of currency
+	 *   decimal => int    Number of decimal places
+	 *
+	 * @param  string $the_currency The desired currency, e.g. 'USD'.
+	 * @return ?array               Currency object or null if not found.
+	 */
+	private static function get_currency( $the_currency ) {
+		$currencies = array(
```

Comment:
Looks like we're missing `MYR` (Malaysian ringgit) here and `INR` (India rupee) is extraneous.

Reply:
Resolved in https://github.com/Automattic/jetpack/pull/10555/commits/2ef9d9065893b33403dbb527dda61cae28f3188b and https://github.com/Automattic/jetpack/pull/10555/commits/8738f686c4ddc22e1767deb630a22afd3ff3e433